model/SearchEngine/Contract/SearchInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
```

Comment:
Suggesting to quickly fix this small PSR issue as part of this PR

```suggestion
<?php

/**
```